jp2_online/static/js/captura/captura_integrantes.js

```diff
@@ -0,0 +1,165 @@
+$(document).ready(function() {
+  $('#tablaCapturista').dataTable();
+  $('#id_fecha_de_nacimiento').datepicker({
+    changeMonth: true,
+    changeYear: true}
+  );
+  // hide inputs from creation form
+  $('#modal_create_integrante #_id_escuela').hide();
+  $('#modal_create_integrante #_id_numero_sae').hide();
+  $('#modal_create_integrante #_id_relacion').hide();
```

Comment:
Not necessary for code review. But you can call only the id's with "_" prefix #_id_escuela, even if they are inside the modal they should be unique in the page.